api/nodes/views.py

```diff
@@ -610,7 +610,7 @@ class NodeDraftRegistrationsList(JSONAPIBaseView, generics.ListCreateAPIView, No
     Use DraftRegistrationsList endpoint instead.
     """
     permission_classes = (
-        IsAdminContributor,
+        IsContributorOrAdminContributor,
```

Comment:
This needs to be something like IsWriteContributor, which I don't think exists.

Reply:
Going to be split out to another ticket